Telegram bot

```diff
@@ -0,0 +1,256 @@
+import numpy as np
+import math
+import telebot
+from telebot import types
+from peewee import *
+from datetime import datetime
+import threading, schedule
+import time as vremya
+
+
+bot = telebot.TeleBot('5245796353:AAHKpjAaha_JY9UDNLgSKjWSShLjlKaTf8I');
+
+db = SqliteDatabase('plans.db') #чудовищно и беспощадно создадим под все эвенты всего одну таблицу
+
+class Plans(Model):
+    uid = IntegerField()
+    event = CharField()
+    date = CharField()
+    time = CharField()
+    notification = BooleanField()
+    flag = IntegerField() #this one is for controlling the fact of sending notification
+
+    class Meta:
+        database = db  
+Plans.create_table()
+        
+notifications=True
```

Comment:
You don't need spaces around `=` when it's a keyword argument or a parameter (e.g.: `@bot.message_handler(commands=['start'])` is correct). But when it's a variable, you should write `notifications = True` according to PEP8

PyCharm may help you to avoid codestyle problems, for example